src/components/reusable/errorComponent/errorComponent.ts

```diff
@@ -0,0 +1,63 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import ErrorComponentScss from './errorComponent.scss';
+
+import '@kyndryl-design-system/shidoka-foundation/components/button';
+
+/**
+ * Error component.
+ * @slot image - Slot for error image.
+ * @slot description - Slot for error description.
+ */
+
+@customElement('kyn-error-component')
```

Comment:
Having "component" in the name seems redundant. What about just "error", "errorPage", "errorDetails", or similar?

Reply:
done.